app/assets/stylesheets/components/_map.scss

```diff
@@ -1,30 +1,61 @@
-@import 'maps/filters';
-@import 'maps/filter';
-@import 'maps/toggler';
-@import 'maps/v-map-baselayer-controls';
+@import "maps/v-map-baselayer-controls";
+@import "maps/v-map-filter";
+@import "maps/v-map-filters";
+@import "maps/v-map-header";
+@import "maps/v-map-toggler";
 
 .map {
   &--main {
     @include gutters;
-    
-    .map {
 
+    .map {
       &__mapbox {
-        height: rem-calc(700);
+        height: rem-calc(360);
+        @include breakpoint($medium) {
+          height: rem-calc(700);
+        }
+      }
+    }
+
+    // ---- BEGIN ONLY SHOW IMMEDIATE HEADER AT MOBILE BREAKPOINT ---- //
+    @include breakpoint($medium) {
+      & > .v-map-header {
+        display: none;
+      }
+    }
+    // ---- ENDOF ONLY SHOW IMMEDIATE HEADER AT MOBILE BREAKPOINT ---- //
+
+    // --------- BEGIN VMAPFILTERS GROUP --------- //
+    @include breakpoint($medium) {
+      position: relative;
+    }
+
+    .v-map-filters {
+      @include breakpoint($medium) {
+        position: absolute;
+        width: 33.33%;
+        z-index: 1;
+
+        top: rem-calc(42);
+        left: $gutter-medium * 2;
```

Comment:
We have some conventions on the order of styles, not sure if you've seen this list before: https://github.com/unepwcmc/wiki/wiki/Styling

Reply:
Will adapt!

Reply:
Cheers!